src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -830,7 +830,7 @@ def load_arguments(self, _):
 
     with self.argument_context('aks trustedaccess rolebinding create') as c:
         c.argument('roles', help='comma-separated roles: Microsoft.Demo/samples/reader,Microsoft.Demo/samples/writer,...')
-        c.argument('source_resource_id', options_list=['--source-resource-id', '-s'], help='The source resource id of the binding')
+        c.argument('source_resource_id', options_list=['--source-resource-id', '-r', c.deprecate(target='-s', redirect='--source-resource-id', expiration='2.47.0', hide=True)], help='The source resource id of the binding')
```

Comment:
I suggest removing `expiration='2.47.0'`, because we can only release breaking change in the breaking change release *(Build or Ignite sprint)*, If the setting of `expiration` is inaccurate, the breaking change may be released in advance
```suggestion
        c.argument('source_resource_id', options_list=['--source-resource-id', '-r', c.deprecate(target='-s', redirect='--source-resource-id', hide=True)], help='The source resource id of the binding')
```

Reply:
sure, updated.